codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -119,8 +125,47 @@ If there are obsolete binaries in entitlements configuration files, please delet
 update these file paths accordingly.
 ''';
 
+  /// Extract credentials needed for code sign app.
+  ///
+  /// Assume the passwords exist with the file path /tmp/passwords.txt.
+  Future<void> readPasswords(FileSystem fileSystem) async {
+    String passwordsFilePath = '/tmp/passwords.txt';
```

Comment:
we should read this path from a command line argument, not just hardcode one. consider either another system could have written to this path, or the operating system could delete it.

Reply:
good point, thanks for the advice Christopher!